### ISSUE TITLE
Crash on export [Windows]

### ISSUE THREAD

**Issue by apraetor** [OpenShot/openshot-qt#849]
**System Details:**
* Operating System / Distro: Windows 10
* OpenShot Version: 2.3.4
Log: [openshot-qt.zip](https://github.com/OpenShot/openshot-qt/files/1246406/openshot-qt.zip)
Note: This is the same bug as #207 
Crash occurs anywhere between 1% and 99%, though most commonly between 40% and 60%. It never successfully finishes an export. Please let me know what I can do to help further diagnose this issue. It's been around for over a year now.

**Issue Description and steps to reproduce:**
1. Create a project
2. Start export.

**Comment by DylanC** [comment]
@apraetor Thanks for testing on the latest version. I will come back to test this myself.

**Comment by apraetor** [comment]
@DylanC  Is closing an active ticket really the best solution? It makes it appear as if this bug is new, when it has been extant for over a year now.

**Comment by DylanC** [comment]
@apraetor - It is more important to know if the issue exists in the latest version of OpenShot. Knowing how long the problem has been around doesn't give us anything really.

**Comment by apraetor** [comment]
It breaks the sense of continuity, though. This issue has existed since the first betas went out to backers.

Can you please tell me how I can help get you the data needed to identify the cause? This occurs 100% of the time, so surely there must be some clue to point the way? I will give you a complete clone of my C: drive, I really don't care. Whatever it takes.

**Comment by apraetor** [comment]
I would mention that knowing the age does give you one piece to consider: have Windows installs remained flat or grown in that time? What percent of users on Windows have a complete inability to use OpenShot successfully? 

I did a complete rebuild of Windows a couple of months ago, not because of OpenShot, but I had hoped it would resolve the issue. It didn't. I stayed silent and waited for it to be addressed, which I guess was the wrong decision. 

If Windows is still a target platform then a bug negatively impacting many users, for a long time, ought to be a priority -- certainly over other less serious Windows-specific bugs.
